app/styles/front.scss

```diff
@@ -637,10 +637,7 @@ $autre-lighter: #cce4e7;
     color: #888;
   }
 
-  .montant::after {
-    @extend .fa;
-    @extend .text-warning;
-    content: $fa-var-warning;
+  i {
```

Comment:
This is a ` .resultats-accordion-needs-n-2 i` selector, so maybe not as big you think, but I agree still too big, I just realised there are some other `i` inside the block.

The point of this block is to vertically center the warning sign:

Without it : 
![image](https://cloud.githubusercontent.com/assets/11834997/17208490/43b80306-54b9-11e6-976d-21995771988c.png)

With it : 
![image](https://cloud.githubusercontent.com/assets/11834997/17208547/78e09c5a-54b9-11e6-89de-1a86a8a4036e.png)